webpack/assets/javascripts/jquery.ui.custom_spinners.js

```diff
@@ -110,6 +110,11 @@ $(() => {
 
 export function initAll(selection = null) {
   // might be called with Event-argument, if used as EventHandler
+  window.tfm.tools.deprecate(
+    'initAll()',
+    'react memory and cpu components',
```

Comment:
Can you add  `cpu` to the `.eslintrc` file?

Reply:
I added